### PR TITLE
Expandable list of information in forecasts

### PR DESCRIPTION
**Pull request by patrykraj**
I created expandable list of additional information to hourly & long term forecasts by clicking on given hour/day.

### CODE REVIEW

File: src/components/forecast/Day.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+export default Day;
+
+const DayElement = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    background: rgba(255,255,255,.2);
+    margin: 10px 0;
+    padding: 0 1rem;
```
Comment (ivan-ivan) [inline src/components/forecast/Day.js:46]: I noticed that some of the font-size values are in rem and some have pixels. Can you please use all px for it across the app? Or at least in this PR would be great

**Review by ivan-ivan — CHANGES_REQUESTED** [review]
Looks good, left some comments

File: src/components/forecast/Day.js (hunk 1 above)
Comment (patrykraj) [reply src/components/forecast/Day.js:46]: I changed all font-size values to pixels.

**Review by ivan-ivan — APPROVED** [review]
👍

**Pull request merged**